python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -129,6 +170,37 @@ def __from_edgelist(
         self.source_columns = source
         self.destination_columns = destination
 
+        self.compute_renumber_edge_list(
+            transposed=False, legacy_renum_only=True
+        )
+
+        self.properties.renumbered = self.renumber_map.implementation.numbered
+        ddf = self.edgelist.edgelist_df
+
+        num_edges = len(ddf)
+        edge_data = get_distributed_data(ddf)
+        src_col_name = self.renumber_map.renumbered_src_col_name
+        dst_col_name = self.renumber_map.renumbered_dst_col_name
+        graph_props = GraphProperties(is_multigraph=self.properties.multi_edge)
```

Comment:
the parameter `is_symmetric` is missing in the `GraphProperty`. For instance it needs to be sets to `True` for `core_number` or `triangle_count` otherwise it causes a crash somewhere in the C/C++ API. I think it should be set to `self.properties.directed`.